docs/fsharp/using-fsharp-on-azure/package-management.md

```diff
@@ -14,25 +14,45 @@ Obtaining packages for Azure development is easy when you use a package manager.
 If you're using [Paket](https://fsprojects.github.io/Paket/) as your dependency manager, you can use the `paket.exe` tool to add Azure dependencies. For example:
```

Comment:
Personally, I think that we should remove the references to the stand-alone `paket.exe` and focus only on the .NET Core scenarios.   Even if we're targeting the legacy .NET Framework or other platform, developers would be encouraged to have a the .NET SDK installed, which includes the `dotnet` CLI.

@jongio: I'd appreciate your thoughts.

Reply:
Agree.